source/server-discovery-and-monitoring/server-monitoring.rst

```diff
@@ -158,6 +156,10 @@ be able to proceed anyway.
 Clients update the topology from each handshake
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+When a monitor check creates a new connection, it MUST perform the
+`connection handshake`_ and MUST use the handshake's ismaster response to
+create the next ServerDescription and update the topology.
```

Comment:
I wonder if this is somewhat prescriptive, can we just say "use the handshake's response to update the ServerDescription"

Reply:
I understand and I'd like to rephrase this. For more background I think we do want to mandate that the Monitor's check only runs a single isMaster command in this case. We want to prevent drivers from running an extra isMaster directly after the handshake. In other words, the handshake MUST satisfy the initial check (or any subsequent check that creates a new connection). Perhaps that wording is better?

> When a monitor check creates a new connection, the `connection handshake`_ response MUST be used to satisfy the check and update the topology.

Reply:
great, I like it